.travis.yml

```diff
@@ -61,6 +61,17 @@ matrix:
           packages:
             - gcc-multilib
             - libgmp-dev:i386
+    # clang with memory sanitizer
+    - compiler: clang
+      # --disable-openssl-tests because openssl uses uninitialized memory. ASM
+      # and BIGNUM are disabled because clang memory sanitizer does not work
+      # with inline assembly (https://clang.llvm.org/docs/MemorySanitizer.html).
+      # The memory sanitizer is instructed to exit with a different exit code
+      # using MSAN_OPTIONS. This is because the default exit code is 77 - the
```

Comment:
Wow, how long did it take you to figure this out?

Reply:
Haha, not long luckily. The automake doc is pretty clear about exit code 77 being the cause of skipped tests.